tests/rest/RestJapaneseTest.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+namespace Keyman\Site\com\keyman\api\tests\rest;
+
+require_once(__DIR__ . '/../../tools/base.inc.php');
+require_once(__DIR__ . '/../TestUtils.inc.php');
+require_once(__DIR__ . '/../TestDBBuild.inc.php');
+require_once(__DIR__ . '/RestTestCase.php');
+
+use Keyman\Site\com\keyman\api\tests\TestUtils;
+
+final class RestJapanesePinyinTest extends RestTestCase
```

Comment:
1. I note the test parameter is `kana` instead of `pinyin`. Would the class be named RestJapaneseKanaTest?

2. What happens if the parameters are swapped?
e.g. chinese_pinyin.php?kana=...
japanese.php?py=ta...

Reply:
1. or maybe just RestJapaneseTest to match the filename?